src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -16,8 +16,12 @@ Author: Diffblue Ltd.
 #include <util/std_code.h>
 #include <util/std_expr.h>
 #include <util/string2int.h>
+#include <util/string_utils.h>
 #include <util/ui_message.h>
 
+#include <algorithm>
+#include <set>
```

Comment:
These includes aren't actually needed in this, but a later commit. Should be moved